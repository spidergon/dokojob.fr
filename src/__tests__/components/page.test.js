import { render, screen } from '@testing-library/react';
import Page from '@components/page';

const MyPage = () => (
  <Page>
    <h1>The title</h1>
    <p>The first paragraph of my page</p>
    <p>The second</p>
  </Page>
);

it('renders Page component unchanged', () => {
  const { container } = render(<MyPage />);

  expect(container).toMatchSnapshot();
});

it('renders Page component Children correctly', () => {
  const { container } = render(<MyPage />);
  const heading = screen.getByRole('heading', {
    name: /The title/i,
  });

  expect(heading).toBeInTheDocument();
});
