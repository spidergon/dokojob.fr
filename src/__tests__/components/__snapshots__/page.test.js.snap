// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Page component unchanged 1`] = `
<div>
  <div
    class="container page"
  >
    <h1>
      The title
    </h1>
    <p>
      The first paragraph of my page
    </p>
    <p>
      The second
    </p>
    <style
      global="true"
      jsx="true"
    >
      
        .page {
          margin-top: 2em;
          padding: 0 1em;
        }
        .page h1 {
          font-size: 2rem;
        }
        .page h2 {
          font-size: 1.25rem;
          margin-top: 2em;
        }
        .page p {
          margin: 1em 0 0.5em;
        }
        .page ul {
          margin: 1em 0;
          padding-left: 2.5em;
        }
      
    </style>
  </div>
</div>
`;
